src/stores/manager/service.go

```diff
@@ -125,64 +121,58 @@ func (m *BaseManager) loadAll() {
 	}
 }
 
-func (m *BaseManager) GetSecretStore(ctx context.Context, name string, userInfo *authtypes.UserInfo) (secrets.Store, error) {
+func (m *BaseManager) GetSecretStore(ctx context.Context, storeName string, userInfo *authtypes.UserInfo) (stores.SecretStore, error) {
 	m.mux.RLock()
 	defer m.mux.RUnlock()
 
-	if storeBundle, ok := m.secrets[name]; ok {
-		if store, ok := storeBundle.store.(secrets.Store); ok {
+	if storeBundle, ok := m.secrets[storeName]; ok {
+		if store, ok := storeBundle.store.(stores.SecretStore); ok {
 			policies := m.policyManager.UserPolicies(ctx, userInfo)
-			resolvr, err := policy.NewResolver(policies)
-			if err != nil {
-				return nil, err
-			}
-			return secretsconnector.NewSecretConnector(store, resolvr, storeBundle.logger), nil
+			_, _ = policy.NewResolver(policies)
+			return secretsconnector.NewSecretConnector(store, m.db.Secrets(storeName), storeBundle.logger), nil
 		}
 	}
 
 	errMessage := "secret store was not found"
-	m.logger.Error(errMessage, "store_name", name)
+	m.logger.Error(errMessage, "store_name", storeName)
 	return nil, errors.NotFoundError(errMessage)
 }
 
-func (m *BaseManager) GetKeyStore(ctx context.Context, name string, userInfo *authtypes.UserInfo) (connectors.KeysConnector, error) {
+func (m *BaseManager) GetKeyStore(_ context.Context, storeName string, _ *authtypes.UserInfo) (stores.KeyStore, error) {
```

Comment:
don't we need the context for the connector? What about the user info?

Reply:
since there is not async calls made within the execution of this method and logging does not depend on ctx, there is not need for context. We added context to all our methods just for consistency and to prevent method signature changes in case context is required in the future